FreeRTOS/FreeRTOS
# Prune excluded dirs in autorelease

Description
-----------
Avoid searching `.git` directories in `os.walk`

Test Steps
-----------


Related Issue
-----------
n/a


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
